### ISSUE TITLE
undefined method Illuminate\Events\Dispatcher::fire() 

### ISSUE THREAD

**Issue by alexcagliari87** [gerardojbaez/laraplans#48]
Hi,
when i try this code:

```
$user = Auth::user();
        
$plan = Plan::where('name','Test')->first();
        
$user->newSubscription('lorem ipsum', $plan)->create();
```

i get an exception:

> Call to undefined method Illuminate\Events\Dispatcher::fire() {"userId":1,"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to undefined method Illuminate\\Events\\Dispatcher::fire() at /var/www/html/xxx/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:237) ...........

What can we do to solve this problem? Is fire a deprecated method?
I'm using laravel 5.8.
Thanks

**Comment by Yahav** [comment]
**The 'fire' Method**
_The fire method (which was deprecated in Laravel 5.4) of the Illuminate/Events/Dispatcher class has been removed. You should use the dispatch method instead._

**Comment by rifan75** [comment]
just change file src\Laraplans\Models\PlanSubscription.php line 87,  Event::fire to Event:dispatch.
I already aplly pull request about this issue

**Comment by siarheipashkevich** [comment]
Any updates?

**Comment by samtoya** [comment]
Any updates on this please? I need to deploy my application to production and it relies on this package.

Thanks guys.

**Comment by siarheipashkevich** [comment]
@gerardojbaez any updates?

**Comment by gerardojbaez** [comment]
I've just released v3.0.0 which addresses this issue, along with support for L5.8. Feel free to open a new issue should anyone have any issues.
